spec/helpers/format_content_helper_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe FormatContentHelper do
```

Comment:
Also if you made it a component you could have a nice component spec with `render_inline` and `expect(page)...` etc 🙂